superset/connectors/sqla/models.py

```diff
@@ -1073,7 +1073,7 @@ def mutator(df: pd.DataFrame) -> None:
     def get_sqla_table_object(self) -> Table:
         return self.database.get_table(self.table_name, schema=self.schema)
 
-    def fetch_metadata(self) -> None:
+    def fetch_metadata(self, dttm_config={}) -> None:
```

Comment:
Using mutable default args in python is a common gotcha https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments